string container_id
duration pouring_duration
---
Result result
---
string status
